src/bump-stdlibs.jl

```diff
@@ -107,6 +107,7 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                 stdlib_current_commit_in_upstream_short = strip(read(`git rev-parse --short $(stdlib_current_commit_in_upstream)`, String))
                 assert_string_startswith(stdlib_current_commit_in_upstream, stdlib_current_commit_in_upstream_short)
                 run(`git fetch --all --prune`)
+                stdlib_version = Base.VersionNumber(Base.parsed_toml("Project.toml")["version"])
```

Comment:
FWIW, I'd be fine with adding TOML as a dependency of this package, and then you could use TOML (instead to needing to use `Base.parsed_toml`).

Reply:
What's wrong with `Base.parsed_toml`?